spec/app/tournament/tournament.spec.js

```diff
@@ -0,0 +1,121 @@
+import {Tournament} from "../../../app/tournament/tournament";
+import {Player} from "../../../app/tournament/player";
+
+describe('class Tournament', function() {
+
+    var spiderman = new Player("Spiderman", 1);
+    var batman = new Player("Batman", 3);
+    var theThing = new Player("The Thing", 5);
+
+    var tournament;
+
+    beforeEach( function() {
+        tournament = new Tournament();
+    });
+
+    function setMockedTournament(players) {
+        while (players--) {
+            tournament.addPlayer(spiderman);
+        }
+    }
+
+    it('should return false when searching for a non-enrolled player', function() {
+        expect( tournament.existsPlayer(spiderman) ).toBeUndefined();
+    });
+
+    it('should return a validation error when validating a player who is already enrolled', function() {
+        tournament.addPlayer(spiderman);
+        var validationResult = tournament.validatePlayer(spiderman);
+
+        expect( validationResult.error ).toEqual("Player already enrolled");
+    });
+
+    it('should return an unsuccessful validation when validating a player who is already enrolled', function() {
+        tournament.addPlayer(spiderman);
+        var validationResult = tournament.validatePlayer(spiderman);
+
+        expect( validationResult.success ).toBe(false);
```

Comment:
This can be on previous test, so we avoid duplicating code. Same with the next 2 tests.